### DISCUSSION TITLE
Install payloadcms on plesk server

### DISCUSSION THREAD

**Discussion by Rar9** [payloadcms/payload#1100] [Q&A] [answered]
How to install payloadcms  and a ubuntu 22.04 plesk server?

Do I install mongodb via docker? 
https://support.plesk.com/hc/en-us/articles/115004540294-How-to-install-MongoDB-database-server-in-Plesk-

How to start the app via node.js

**Comment by denolfe** [comment, marked as answer]
Hey @Rar9 , you could likely get a better answer for this elsewhere on the internet, since it's more Ops-related than Payload specifically. I would suggest searching for how to deploy a node application with mongodb using docker-compose as a good starting point.

**Reply by Rar9** [reply]
It's a feature request,  so people are getting easy a guide how to install payloadcms on a plesk environment. 

No quide = no new user as they will turn to strapi /directus....

**Reply by Rar9** [reply]
Basically you can add nginx basic config to the list

**Reply by MarkAtOmniux** [reply]
> It's a feature request, so people are getting easy a guide how to install payloadcms on a plesk environment.
> 
> No quide = no new user as they will turn to strapi /directus....

Ironically I moved from strapi to payload because I hated it. But I agree, Docker documentation would be helpful

**Reply by Rar9** [reply]
I´m currently on Directus due to the missing guide... but would love to test payloadcms as strapi ways also not my thing

**Reply by Rar9** [reply]
I got a bit further

https://discord.com/channels/967097582721572934/1022136604443299840/1022385771031830548
